hack/deploy-custom-nto.sh

```diff
@@ -31,7 +32,14 @@ cvo_scale() {
 
 nto_deploy_custom() {
   oc project openshift-cluster-node-tuning-operator
-  oc patch deploy cluster-node-tuning-operator -p '{"spec":{"template":{"spec":{"containers":[{"env":[{"name":"CLUSTER_NODE_TUNED_IMAGE","value":"'$IMAGE'"}],"name":"cluster-node-tuning-operator"}]}}}}'
+  cd "${WORKDIR}" || { echo "failed to change dir to ${WORKDIR}"; exit; }
```

Comment:
Errors should go to `stderr` and we should not exit 0 when there is a failure. Let's exit 1.